packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -147,6 +148,20 @@ export class LoaderContainerTracker implements IOpProcessingController {
         // REVIEW: do we need to unpatch the loaders?
     }
 
+    /**
+     * Ensure all tracked containers are synchronized
+     */
+    public async ensureSynchronized(...containers: IContainer[]) {
+        await this.processSynchronized(-1, ...containers);
```

Comment:
how are negative numbers handled by the timeout task? why not just pass undefined?